src/rustdoc-json-types/lib.rs

```diff
@@ -258,19 +258,19 @@ pub struct Constant {
 ///              ^^^^^^^^^^  ^^^^^^^^^^^^^^^
 /// ```
 #[derive(Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize)]
-pub struct TypeBinding {
+pub struct AssocItemConstraint {
```

Comment:
Why this change?

Reply:
a "type binding" is a confusing name, and frankly outdated, since nowadays it can also constrain a constant, and it only ever operates on associated items, by constraining them